### ISSUE TITLE
SassTask mangles the CLI command depending on attribute order

### ISSUE THREAD

**Issue by GregJPreece** [phingofficial/phing#1206]
**PHP version:** 7.3.11
**Phing version:** 2.16.1

Ran into a very odd issue when using Phing to build my Sass sheets in a new project. When the Sass target is constructed like this, everything works fine:

```
        <sass useScssphp="true" 
              path="theme/base/scss/console/"
              input="theme/base/scss/console/console.scss"
              output="theme/base/css/console-base.min.css"
              compressed="true"
              keepsubdirectories="false">
    </sass>
```

However, previously the attributes were in the order below:

```
        <sass useScssphp="true" 
              compressed="true"
              path="theme/base/scss/console/"
              input="theme/base/scss/console/console.scss"
              output="theme/base/css/console-base.min.css"
              keepsubdirectories="false">
        </sass>
```

This leads to the command failing due to a malformed Sass call, as shown in the output below.

```
     [sass] Process file
     [sass] Executing: sass  --style compressed--load-path theme/base/scss/console/ theme/base/scss/console/console.scss theme/base/css/console-base.min.css

BUILD FAILED
```

As you can see, the `--style` and `--load-path` arguments have been run together without whitespace. I did a little experimentation and noticed that the `compressed` attribute can be replaced with `expand` or `compact`, and the error will still be triggered. However, the scssphp exclusive `crunched` attribute works normally.

I suspect this is a very minor fix. If I find time, I'll go hunting myself.

**Comment by kenguest** [comment]
Fixed in PR #1207.

**Comment by siad007** [comment]
... merged. 😄
